project/api/models.py

```diff
@@ -31,8 +31,14 @@ class Nation(models.Model):
     )
     description = models.TextField(help_text="Flavor text, brief history, etc.",
                                    blank=True)
-    wikipedia = models.URLField(help_text="Link to the Wikipedia article for this nation",
-                                blank=True)
+    links = ArrayField(
+        models.URLField(),
+        null=True,
+        blank=True,
+    )
+    references = ArrayField(
+        models.TextField(max_length=150),
```

Comment:
This is probably fine for now, but should definitely go into its own model at some point.

Or maybe we create a separate model now already?

Reply:
I had a look at https://en.wikipedia.org/wiki/Wikipedia:Citation_templates. It might actually be too complicated if we want to do it well, this could actually be a whole db on itself.

Maybe we could use Wikipedia's syntax to manage citations? So add a regex check like [this one](https://github.com/kenshiro-o/wikipedia-js/blob/master/lib/converter/wikiToHTMLConverter.js)? What do you think?

Reply:
Neat, I'd be inclined to keep it SQL though. Maybe I could write a short script to translate each medium's citation fields on Wikipedia to Django models and respective fields.

Reply:
Sounds good.